app/models/user.rb

```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :bookings
+  has_many :mesages, through: :bookings ### whith through or not
+  has_many :reviews, through: :bookings ### ?
+
+
+  validates :name, presence: true, uniqueness: true
```

Comment:
validate email (presence, uniqueness) and password (presences). name - just presence